app/models/user.js

```diff
@@ -20,6 +20,10 @@ module.exports = (sequelize, DataTypes) => {
       password: {
         allowNull: false,
         type: DataTypes.TEXT
+      },
+      role: {
+        allowNull: false,
+        type: DataTypes.STRING
```

Comment:
Here you could add a default value for when users are created, you could also add the values that are allowed for this column (ADMIN or REGULAR).